spec/60-trace-id-format.md

```diff
@@ -1,20 +1,52 @@
 ## Considerations for trace-id field generation
 
-A vendor SHOULD generate globally unique values for `trace-id`. Many unique
-identification generation algorithms create IDs where one part of the value is
-constant (often time- or host-based), and the other part is a randomly generated
-value. Because tracing systems may make sampling decisions based on the value of
-`trace-id`, for increased interoperability vendors MUST keep the random part of
-`trace-id` ID on the left side.
-
-When a system operates with a `trace-id` that is shorter than 16 bytes, it
-SHOULD fill-in the extra bytes with random values rather than zeroes. Let's say
-the system works with an 8-byte `trace-id` like `3ce929d0e0e4736`. Instead of
-setting `trace-id` value to `0000000000000003ce929d0e0e4736` it SHOULD generate
-a value like `4bf92f3577b34da6a3ce929d0e0e4736` where `4bf92f3577b34da6a` is a
-random value or a function of time and host value.
-
-**Note**: Even though a system may operate with a shorter `trace-id` for
-[distributed trace](https://w3c.github.io/trace-context/#dfn-distributed-traces)
-reporting, the full `trace-id` MUST be propagated to conform to the
-specification.
+This section suggests some best practices to consider when platform or tracing
+vendor implement `trace-id` generation and propagation algorithms. These
+practices will ensure better interoperability of different systems.
+
+### Uniqueness of `trace-id`
+
+The value of `trace-id` SHOULD be globally unique. This field is typically used
+for unique identification of a <a>distributed trace</a>. It is common for
+<a>distributed traces</a> to span various components, including, for example,
+cloud services. Cloud services tend to serve variety of clients and have a very
+high throughput of requests. So global uniqueness of `trace-id` is important,
+even when local uniqueness might seem like a good solution.
+
+### Randomness of `trace-id`
+
+Randomly generated value of `trace-id` SDHOULD be preferred over other
+algorithms of generating a globally unique identifiers. Randomness of `trace-id`
+addresses some [secutiry](#security-considerations) and [privacy
+concerns](#privacy-considerations) of exposing unwanted information. Randomness
+also allows tracing vendors to base sampling decisions on `trace-id` field value
+and avoid propagating an additional sampling context.
+
+As shown in the next section, it is important for better inter-operability with
+some existing systems, for `trace-id` to carry "uniqueness" and "randomness" in
+the right part of the `trace-id`.
```

Comment:
swap parts of the sentence

As shown in the next section, it is important for `trace-id` to carry "uniqueness" and "randomness" in the right part of the `trace-id`, for better inter-operability with some existing systems.